public/apps/configuration/constants.tsx

```diff
@@ -43,6 +43,7 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/ad/detector/write',
   'cluster:admin/opendistro/ad/detectors/get',
   'cluster:admin/opendistro/ad/result/search',
+  'cluster:admin/opendistro/ad/tasks/search',
```

Comment:
can we add preview and search info APIs as well?
`cluster:admin/opendistro/ad/detector/preview`
`cluster:admin/opendistro/ad/detector/info`

Reply:
Sure, will add it. Send out PR earlier to check if build action can pass. Quite slow to setup local dev environment.

Reply:
Sure thanks @ylwu-amzn !